### DISCUSSION TITLE
How to deal with the following error : BR-OPT-00201-0101

### DISCUSSION THREAD

**Discussion by EleonoreLD** [OP-TED/eForms-SDK#438] [Q&A] [answered]
Good afternoon, 

I filled in an F01 form on eNotice2. 
Once all the fields in the form have been filled in, we click on "submit a notice". 
I have the following error (which has a double occurrence because we have two organizations mentioned so two touchpoints) that prevents me from sending the notification : rule|text|BR-OPT-00201-0101.

Looking for the error message associated with the BR-OPT-00201-0101 rule, I find the following message: "Any existing touchpoint should be associated with at least one of the following role/subrole: Additional Info Provider, Doc Provider, Employment, Environmental or Fiscal Legislation information provider, Mediator, Review Information Provider, Review Organization, Tender Evaluator or Recepient, Review Body, Review Requester."

The message appears to refer to the OPT-201 field which is not one of the fields displayed to the user in the form. The OPT-201 field is described as a TouchPoint technical identifier. This identifier is used in the notice to refer to alternative contact details associated with a specific role/Subrole (as TPO-XXX).

I can't figure out how to assign a role to my touchpoints. Has anyone experienced this problem and could you help me out? 

Thank you
Eléonore

**Comment by rouschr** [comment, marked as answer]
This is related to eNotices2, please see: https://enotices2.ted.europa.eu/help
